### ISSUE TITLE
Give regular users access to debug logs w/o adb or running dev versions of app

### ISSUE THREAD

**Issue by mstyp** [nextcloud/android#9844]
### Is your feature request related to a problem? Please describe.
Yes, right now I think it is to difficult for normal users to diagnose issues they might be having with the android app, especially power users relying on the app to get work done.
Requiring the installation of another piece of software (esp software as complicated as [adb](https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat-mag)) to access the logs and diagnose problems is asking to much.

### Describe the solution you'd like
Add the ability to look at logs in the regular nextcloud app the same way users can look at logs in the dev app.

### Describe alternatives you've considered
Maybe offering a "power users" version of the app, for people who rely on the app in the same way that they rely on email or calendar (IE, they rely on it professionally) so they need the ability to diagnose issues, but they also don't want to use / don't know that they should be using the developers version. 

My suspicion is that adding another version of the app is overkill and that it would confuse more people then it helps, but it is an alternative that I think would solve the problem of "I use this app a lot and need it to work for my job, but I'm having a problem and I could use some more information to help fix it"

### Additional context

I use the nextcloud android app every day, and rely on it heavily for work. I do not work in  tech, so everything I learned is self taught. I set it all up by myself, and maintain it by myself, with the only support coming from the community (which is fantastic! very greatful to devs and community)

Because I use it every day I often stumble across problems, but without easy to access logs within the GUI it can be very hard to tell if this is a problem with my setup or with the app code itself. since I maintain all this myself, I usually have to then set aside time to do research, and if I cant find a solution, open an issue here on github (and if its an issue with my setup then my request for help does not really belong on github)

This may sound like a unique edge case, but i dont think it is. I know there are a lot of DIY folks on reddit /r/selfhosting who use and recommend nextcloud, and I am willing to bet that a non trivial % of them are in a similar boat of maintaining a nextcloud server themselves, but also not using the Dev version of the app. And I am willing to bet that either now (or hopefully in the future) are a lot of proffessionals who use nextcloud proffessionaly and who are facing issues and either they or their IT staff would find logs helpful.

**Ultimately I think its important for any Foss project to make it as easy as possible to ask for help.** Its one thing to ask people for information like "what brand / kind of android are you using". Its even another thing to ask "what specific _version_ of android are you using" which most people wont even know how to check, even if they might be able to easily figure it out / google it. Asking users to install a whole new piece of software (even what we might think of as "simple" software) should be considered an absolute last resort to diagnosing an issue.

**Comment by AlvaroBrey** [comment]
I agree that sharing logs should be a bit easier,  specially for users who don't know what logs even are.

The current logging system logs both to a file and to the system log (which `adb` can read). However both are disabled for production builds. There are many reasons why that may be, the most common of which are performance and privacy.

@tobiasKaminsky what do you think about this? Some ideas:

- Add a (maybe hidden) toggle to enable logging even in the production app, while keeping it disabled by default
- Always enable logging to file, just disable logging to adb in production.

**Comment by tobiasKaminsky** [comment]
- [ ] move logs to "community" page to have it visible all time
- [ ] make it possible to copy entire logs via clipboard to e.g. GitHub
- [ ] make it possible to copy text snippets from log view to clipboard

**Comment by SvenLieber** [comment]
I think this is very relevant as seen in the recent 2026 issue https://github.com/nextcloud/android/issues/16550, where a different app variant has to be installed in order to enable debug logs for auto-upload.

I totally understand the wish to avoid unnecessary logging for security reasons (as stated in the linked issue), but more detailed logging could be enabled via the settings while the default is no logging. Then all users (including non tech-savvy) would be empowered to provide needed context to issues.

**Comment by regs01** [comment]
Including F-Droid. 

> Requiring the installation of another piece of software (esp software as complicated as [adb](https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat-mag)) to access the logs and diagnose problems is asking to much.

Most important is that installing additional version you create a fresh environment in which existing bugs might not fire immediately. It is far more reliable to collect logs on prediction version.
